src/exposition/http.rs

```diff
@@ -248,6 +270,75 @@ mod handlers {
         Ok(content)
     }
 
+    pub async fn binary_metadata() -> Result<impl warp::Reply, Infallible> {
+        let mut metadata = Metadata::new();
```

Comment:
Does this ever change during execution and if not, do we want to regenerate it every time? Seems like we could create the metadata once at start of day.

Reply:
I don't think there's a huge win in caching this. It should not change, but it is relatively cheap to produce.